platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/network/connectivity/TcpFactory.java

```diff
@@ -47,4 +49,9 @@ public TcpFactory(@NonNull final SocketConfig socketConfig) {
         SocketFactory.configureAndConnect(clientSocket, socketConfig, hostname, port);
         return clientSocket;
     }
+
+    @Override
+    public void handlePeerListUpdate(List<PeerInfo> peerInfoList) {
```

Comment:
Naming suggestion. How about `updatePeers()`?

Reply:
thing is it doesn't really *have to* update the peers, though it may, depending on implementation. The goal of the method is to allow the factory take action based on the updated list of peers. Still open to name suggestion though but _updatePeers_ *could be* misleading.